samples/drivers/video/capture/src/main.c

```diff
@@ -10,7 +10,13 @@
 #include <zephyr/drivers/display.h>
 #include <zephyr/drivers/video.h>
 
+#ifdef CONFIG_TEST
+#define LOG_LEVEL 4
```

Comment:
Just putting a few links here in case that helped:
- [`LOG_LEVEL_WRN`](https://github.com/zephyrproject-rtos/zephyr/blob/main/include/zephyr/logging/log_core.h#L23)
- [`CONFIG_VIDEO_LOG_LEVEL`](https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/video/Kconfig#L16-L18)

Reply:
Thanks ! I changed `#define LOG_LEVEL 4` to `#define LOG_LEVEL LOG_LEVEL_DBG`.
Here we need LOG_LEVEL_DBG (a.k.a 4) for logs printed on the console otherwise the test cannot pass.